examples/pylab_examples/quiver_demo.py

```diff
@@ -8,76 +8,89 @@
-               fontproperties={'weight': 'bold'})
-axis([-1, 7, -1, 7])
-title('scales with plot width, not view')
+# 2
+plt.figure()
+Q = plt.quiver(X, Y, U, V, units='width')
+qk = plt.quiverkey(Q, 0.9, 0.95, 2, r'$2 \frac{m}{s}$',
+                   labelpos='E',
+                   coordinates='figure',
+                   fontproperties={'weight': 'bold'})
+plt.axis([-1, 7, -1, 7])
+plt.title('scales with plot width, not view')
 
-#3
-figure()
-Q = quiver(X[::3, ::3], Y[::3, ::3], U[::3, ::3], V[::3, ::3],
-           pivot='mid', color='r', units='inches')
-qk = quiverkey(Q, 0.5, 0.03, 1, r'$1 \frac{m}{s}$', fontproperties={'weight': 'bold'})
-plot(X[::3, ::3], Y[::3, ::3], 'k.')
-axis([-1, 7, -1, 7])
-title("pivot='mid'; every third arrow; units='inches'")
+# 3
+plt.figure()
+Q = plt.quiver(X[::3, ::3], Y[::3, ::3], U[::3, ::3], V[::3, ::3],
+               pivot='mid', color='r', units='inches')
+qk = plt.quiverkey(
+    Q,
+    0.5,
+    0.03,
+    1,
+    r'$1 \frac{m}{s}$',
+    fontproperties={
+        'weight': 'bold'})
+plt.plot(X[::3, ::3], Y[::3, ::3], 'k.')
+plt.axis([-1, 7, -1, 7])
+plt.title("pivot='mid'; every third arrow; units='inches'")
 
-#4
-figure()
-M = sqrt(pow(U, 2) + pow(V, 2))
-Q = quiver(X, Y, U, V, M, units='x', pivot='tip', width=0.022, scale=1/0.15)
-qk = quiverkey(Q, 0.9, 1.05, 1, r'$1 \frac{m}{s}$',
-               labelpos='E',
-               fontproperties={'weight': 'bold'})
-plot(X, Y, 'k.')
-axis([-1, 7, -1, 7])
-title("scales with x view; pivot='tip'")
+# 4
+plt.figure()
+M = np.sqrt(np.power(U, 2) + np.power(V, 2))
+Q = plt.quiver(X, Y, U, V, M,
+               units='x',
+               pivot='tip',
+               width=0.022,
+               scale=1 / 0.15)
+qk = plt.quiverkey(Q, 0.9, 1.05, 1, r'$1 \frac{m}{s}$',
+                   labelpos='E',
+                   fontproperties={'weight': 'bold'})
+plt.plot(X, Y, 'k.')
+plt.axis([-1, 7, -1, 7])
+plt.title("scales with x view; pivot='tip'")
 
-#5
-figure()
-Q = quiver(X[::3, ::3], Y[::3, ::3], U[::3, ::3], V[::3, ::3],
-           color='r', units='x',
-           linewidths=(2,), edgecolors=('k'), headaxislength=5)
-qk = quiverkey(Q, 0.5, 0.03, 1, r'$1 \frac{m}{s}$', fontproperties={'weight': 'bold'})
-axis([-1, 7, -1, 7])
-title("triangular head; scale with x view; black edges")
+# 5
+plt.figure()
+Q = plt.quiver(X[::3, ::3], Y[::3, ::3], U[::3, ::3], V[::3, ::3],
+               color='r', units='x',
+               linewidths=(2,), edgecolors=('k'), headaxislength=5)
+qk = plt.quiverkey(Q, 0.5, 0.03, 1, r'$1 \frac{m}{s}$',
+                   fontproperties={'weight': 'bold'})
+plt.axis([-1, 7, -1, 7])
+plt.title("triangular head; scale with x view; black edges")
 
-#6
-figure()
-M = zeros(U.shape, dtype='bool')
-M[U.shape[0]/3:2*U.shape[0]/3, U.shape[1]/3:2*U.shape[1]/3] = True
+# 6
+plt.figure()
+M = np.zeros(U.shape, dtype='bool')
+M[U.shape[0] / 3:2 * U.shape[0] / 3, U.shape[1] / 3:2 * U.shape[1] / 3] = True
```

Comment:
Agreed. In this version, I see the `3:2` grouped together, even though they are separate. If anything, I would instead have each slice on their own lines. This would also keep it under 80 chars.